package.json

```diff
@@ -13,6 +13,7 @@
     "test": "test"
   },
   "scripts": {
+    "prepare": "husky install",
```

Comment:
The [npm prepare script](https://docs.npmjs.com/cli/v7/using-npm/scripts) only runs when in the development environment.